### PR TITLE
Modification of logos and other changes

### PR DESCRIPTION
**Pull request by MaXfAa**
Modification of logos (and colors), addition of descriptions in French and other small changes to the following presences :

1. Bing
2. Google Calendar
3. Google Docs
4. Dominos
5. Google Drive
6. Google Fonts
7. Gmail
8. Google Meet
9. (Facebook) Messenger
10. myCanal
11. ProBot
12. Rythm
13. Skype
14. Zalando

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Not enough to add you as contributor.
Upload to Imgur.
And the logos need to be changed in the Discord Application, and I bet you didnt contact the presence authors to update that... (Cause I am the author of a few of them and got no message)

**Comment by MaXfAa** [comment]
> Not enough to add you as contributor.
> Upload to Imgur.
> And the logos need to be changed in the Discord Application, and I bet you didnt contact the presence authors to update that... (Cause I am the author of a few of them and got no message)
Ok, so:
1. I changed to Imgur
2. I withdrew from "contributor" (again sorry)
And tomorrow would do the following (contact the authors, etc...)

File: websites/P/ProBot/dist/metadata.json [hunk 1]
```diff
@@ -3,18 +3,26 @@
     "name": "NMW03",
     "id": "534099893979971584"
   },
+  "contributors": [
+    {
+      "name": "Maxou",
+      "id": "448518859184406529"
+    }
+  ],
```
Comment (SlowLife1661) [inline websites/P/ProBot/dist/metadata.json:11]: 👀

File: websites/G/Google Meet/dist/metadata.json [hunk 2]
```diff
@@ -3,18 +3,25 @@
     "name": "Drull",
     "id": "244953147154366464"
   },
+  "contributors": [
+    {
+      "name": "Maxou",
+      "id": "448518859184406529"
+    }
+  ],
```
Comment (SlowLife1661) [inline websites/G/Google Meet/dist/metadata.json:11]: 👀

File: websites/P/ProBot/dist/metadata.json (hunk 1 above)
Comment (SlowLife1661) [inline websites/P/ProBot/dist/metadata.json:11]: 👀

File: websites/M/myCANAL/dist/metadata.json [hunk 3]
```diff
@@ -3,24 +3,20 @@
     "name": "Alexx",
     "id": "604182077088202784"
   },
-  "contributors": [
-    {
-      "name": "Bryan",
-      "id": "293775922757238784"
-    }
-  ],
```
Comment (SlowLife1661) [inline websites/M/myCANAL/dist/metadata.json:11]: 👀

File: websites/R/Rythm/dist/metadata.json [hunk 4]
```diff
@@ -1,26 +0,0 @@
...
-  "thumbnail": "https://i.imgur.com/Wjy6N3p.png",
-  "color": "#fa1919",
-  "tags": [
-    "discord",
-    "music",
-    "bot",
-    "free"
-  ],
-  "category": "other"
-}
```
Comment (SlowLife1661) [inline websites/R/Rythm/dist/metadata.json:26]: 🤷‍♂️

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

File: websites/R/Rythm/dist/metadata.json (hunk 4 above)
Comment (SlowLife1661) [reply websites/R/Rythm/dist/metadata.json:26]: Resolve the conflicts.

File: websites/G/Google Meet/dist/metadata.json (hunk 2 above)
Comment (MaXfAa) [reply websites/G/Google Meet/dist/metadata.json:11]: Wtf

**Comment by Bas950** [comment]
> > Not enough to add you as contributor.
> > Upload to Imgur.
> > And the logos need to be changed in the Discord Application, and I bet you didnt contact the presence authors to update that... (Cause I am the author of a few of them and got no message)
> > Ok, so:
> 
> 1. I changed to Imgur
> 2. I withdrew from "contributor" (again sorry)
>    And tomorrow would do the following (contact the authors, etc...)

What about 3.?

**Comment by ririxidev** [comment]
Resolve conflicts, do everything that Bas said then open new pull request.

Thanks,
ririxi (Contact: mail@ririxi.dev, ririxi#2721)

**Pull request closed without merging**